package.json

```
@@ -61,6 +61,11 @@
                 "category": "AHK++",
                 "icon": "./image/build.svg"
             },
+            {
+                "command": "ahk++.compilerGui",
+                "title": "Compiler GUI",
```

Let's keep this as close to the default File Explorer context menu command name as we can for familiarity. We just have to specify "AHK" script since we're in an environment that supports many languages

```suggestion
                "title": "Compile AHK Script (GUI)",
```